tests/fixed.test.js

```diff
@@ -17,7 +17,7 @@ describe('Fixed thread pool test suite ', () => {
   it('Choose worker round robin test', async () => {
     const results = new Set()
     for (let i = 0; i < numThreads; i++) {
-      results.add(pool._chooseWorker().threadId)
+      results.add(pool.chooseWorker().threadId)
```

Comment:
If choose worker is protected how we are accessing it here ? mmmmmm

Reply:
member accessors are just a TS feature that will be removed while transpiling
JS is pure public (not anymore with `#`-private properties, but please don't use them)

Reply:
Ok so in our change log we should not say that are no more accessible but that the _ is removed and we anyway are not intended to be used